lib/msf/core/web_services/servlet/event_servlet.rb

```diff
@@ -22,9 +22,9 @@ def self.get_event
       warden.authenticate!
       begin
         sanitized_params = sanitize_params(params, env['rack.request.query_hash'])
-        data = get_db.events(sanitized_params)
+        data, pagination_total = get_db.events(sanitized_params)
         data = data.first if is_single_object?(data, sanitized_params)
```

Comment:
Just confirming, does this pagination merging approach take into consideration both scenarios for this line? i.e. Handling the scenario of this line running `data = data.first` successfully, and the other scenario where this line doesn't run and data still equals the original `get_db.event(...)` ?